### PR TITLE
Install minio as a subchart of the epinio chart

### PR DESCRIPTION
**Pull request by jimmykarily**
Using this chart: https://github.com/minio/minio/tree/master/helm/minio
which doesn't deploy an operator (which we didn't need anyway).
This removes one dependency from the installer manifest. One more step
closer to single-chart.

Relevant discussion: https://github.com/epinio/epinio/discussions/1190

### CODE REVIEW

File: chart/epinio/Chart.yaml [hunk 1]
```diff
@@ -12,4 +19,4 @@ maintainers:
 name: epinio
 sources:
 - https://github.com/epinio/epinio
-version: 0.3.0
+version: 0.4.0
```
Comment (ldevulder) [inline chart/epinio/Chart.yaml:22]: Done by `updatecli`, has to be removed. Sorry my first thought was false...

File: chart/epinio-installer/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: epinio-installer
-version: 0.4.0
+version: 0.5.0
```
Comment (ldevulder) [inline chart/epinio-installer/Chart.yaml:3]: Done by `updatecli`, has to be removed. Sorry my first thought was false...

Comment (jimmykarily) [reply chart/epinio-installer/Chart.yaml]: done

File: chart/epinio/Chart.yaml (hunk 1 above)
Comment (jimmykarily) [reply chart/epinio/Chart.yaml]: done

**Comment by jimmykarily** [comment]
We can use cert-manager and this helm chart to fix this: https://github.com/epinio/helm-charts/pull/97/files#r806568901

**Review by manno — APPROVED** [review]

**Review by ldevulder — APPROVED** [review]
Testing successfully on my lab. Let's see how how it will work on Public Clouds after merging :-)

**Pull request merged**